android/app-newm/src/main/java/io/newm/screens/profile/ProfileForm.kt

```diff
@@ -68,26 +76,26 @@ fun ProfileForm(
                     style = formTitleStyle
                 )
                 Spacer(modifier = Modifier.height(16.dp))
-                TextFieldWithLabel(
-                    labelResId = R.string.profile_form_password_current_password,
-                    onValueChange = { value -> onProfileUpdated(profile.copy(currentPassword = value)) },
-                    isPassword = true,
-                    textfieldBackgroundColor = Gray23,
-                    placeholderResId = R.string.profile_placeholder_your_password
+                Password(
```

Comment:
Do we need an email code to be able to update password? I assume we don't just double-checking